modules/ROOT/pages/product-versioning.adoc

```diff
@@ -26,6 +26,16 @@ Here is a Bonita Platform version / Technical Id / UI Designer version correspon
 |===
 | Release date | Bonita Platform version | Technical Id | UI Designer version
 
+| 2022-03-07
+| 2021.1-0307
+| 7.12.11
+| 1.12.?
+
+| 2022-02-02
+| 2021.1-0202
+| 7.12.10
+| 1.12.?
+
 | 2021-11-19
```

Comment:
⚠️ according to the release-notes this should be 2021-11-16 ⚠️ 
@NathalieC can I change this? Otherwise, I will change the release notes page for consistency.

Reply:
sure you can @tbouffard ; thi is the correct date for the build

Reply:
done in 1a58792